mmseg/models/necks/upsample_neck.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+from ..builder import NECKS
+
+
+@NECKS.register_module()
+class UpsampleNeck(nn.Module):
```

Comment:
We may rename it to something like `MultiLevelNeck` since we would also perform downsampling.